### ISSUE TITLE
Write description for onboarding sessions

### ISSUE THREAD

**Issue by zzap** [WordPress/Documentation-Issue-Tracker#830]
Onboarding session videos are great and useful, but in a situation like a Contributor Day, watching videos might not be the best way or even possible.

Writing a description for the video in the form of a step-by-step guide would be more helpful in this scenario.

- [ ] [Overview](https://wordpress.tv/2022/06/21/milana-cap-overview-onboarding-for-wordpress-documentation-team/) 
- [ ] [End user documentation](https://wordpress.tv/2022/06/21/milana-cap-end-user-documentation-onboarding-for-wordpress-documentation-team/)
- [ ] [Plugin handbook](https://wordpress.tv/2022/06/21/milana-cap-developer-documentation-plugin-handbook-onboarding-for-wordpress-documentation-team/)
- [ ] [Common APIs handbook](https://wordpress.tv/2022/06/21/milana-cap-developer-documentation-common-apis-handbook-onboarding-for-wordpress-documentation-team/)
- [ ] [Code reference handbook](https://wordpress.tv/2022/06/21/milana-cap-developer-documentation-code-reference-handbook-onboarding-for-wordpress-documentation-team/)
- [ ] [Block editor handbook](https://wordpress.tv/2022/06/30/milana-cap-developer-documentation-block-editor-handbook-onboarding-for-wordpress-documentation-team/)
- [ ] [Themes handbook](https://wordpress.tv/2022/06/30/milana-cap-developer-documentation-themes-handbook-onboarding-for-wordpress-documentation-team/)
- [ ] [Documentation team handbook](https://wordpress.tv/2022/06/21/milana-cap-contributor-documentation-documentation-team-handbook-onboarding-for-wordpress-documentation-tea/)
- [ ] [Github related roles](https://wordpress.tv/2023/02/08/github-related-roles-onboarding-for-wordpress-documentation-team/)
- [ ] Advanced Administration handbook - video doesn't exist  😱

Descriptions should live in [Documentation team handbook](https://make.wordpress.org/docs/handbook/) under Get involved - create a new page, "Onboarding sessions".

**Comment by zzap** [comment]
Heads up @WordPress/docs-issues-coordinators, we have a new issue open. Time to use 'em labels.

**Comment by zzap** [comment]
# Onboarding

What skills do I need?
- English language

How much experience with WordPress do I need?
- Anything in between none and a lot. The best documentation is written when you do things for the first time. Also, if you don't have any experience with WordPress, then you are the best person to validate existing documentation and point to possible holes in it.

Which accounts do I need?
- [WordPress.org account](https://login.wordpress.org/register).
- [WordPress Slack account](https://make.wordpress.org/chat/) (wordpress.org account is needed).
- [GitHub account](https://github.com/signup) – connect [your GitHub account to your WordPress.org profile](https://make.wordpress.org/core/2020/03/19/associating-github-accounts-with-wordpress-org-profiles/).

Places where the Docs team operates:
- [Blog](https://make.wordpress.org/docs/) – for meeting agendas and summaries (and anything related to the Docs team).
- [Slack channel #docs](https://wordpress.slack.com/archives/C02RP4WU5) – where meetings are happening (and all communication regarding the team itself).
- [GitHub repository](https://github.com/WordPress/Documentation-Issue-Tracker) – where issues for all documentation are reported, discussed and worked on.
- [Handbook](https://make.wordpress.org/docs/handbook/) – how to contribute to the Documentation team (it’s a bit out of date).
- [Style guide](https://make.wordpress.org/docs/style-guide/) – for how to write WordPress documentation.

When and where are meetings happening?
Meetings happen in [#docs](https://wordpress.slack.com/archives/C02RP4WU5) Slack channel. Days and timing can be found in several places: The Welcome box (on the top of every page in this handbook or blog), the sidebar on the [blog page](https://make.wordpress.org/docs/), [global meeting calendar](https://make.wordpress.org/meetings/).

What type of meetings does the Docs team hold?
- Regular meetings - in the form of chat in the Slack channel. These meetings are for project updates (showing off what you've been working on in documentation) and discussing any topics relevant to the team.
- Issues triage - in the form of chat in the Slack channel. These meetings are for discussing [GitHub issues](https://github.com/WordPress/Documentation-Issue-Tracker/issues): assigning, re-assigning, unblocking, helping assignees etc.
- Online Contributor Days - in the form of conference calls over Zoom. These are 3 hours long, and we use them to work on GitHub issues, help new contributors get started, and discuss any problems contributors come across while working on these issues.

**Comment by zzap** [comment]
# Updating screenshots for WordPress end-user documentation

Recording: https://wordpress.tv/2022/03/04/update-screenshots-in-wordpress-documentation/

## What do I need?
- Fresh WordPress install - [InstaWP](https://instawp.com/)
- Screenshot tool -
